package.json

```diff
@@ -6,7 +6,7 @@
     "@babel/polyfill": "^7.7.0",
     "@datapunt/asc-assets": "^0.17.7",
     "@datapunt/asc-core": "^0.17.7",
-    "@datapunt/asc-ui": "^0.17.7",
+    "@datapunt/asc-ui": "^0.17.8-alpha.12",
```

Comment:
This is the canary version. Why would you upgrade to this version if the latest stable is still `0.17.7`?

Reply:
I tryed to use a control from that version, but not anymore. I'll change it to the stabile version.